napalm_nxos/nxos.py

```diff
@@ -204,7 +204,59 @@ def get_facts(self):
                 iface_list.append(intf)
 
         return results
+  
+    def get_fex_information(self):
+        item_list = list()
+
+        fex_command = 'show fex'
```

Comment:
Fabric Extenders for Nexus 5k/2k.
DS# sh fex
  FEX         FEX           FEX              FEX              Fex
Number    Description      State            Model            Serial
------------------------------------------------------------------------
102      ATL1-AS3P                Online   N2K-C2248TP-E-1GE   SSIxxxxxxxx
etc.
etc
etc.